### PR TITLE
Feature/longrunjobdoc

### PR DESCRIPTION
**Pull request by zfengms**
add documentation and sample for long running job feature

### CODE REVIEW

File: docs/31-long-running-job.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Synchronous
+- Asynchronous
+
+Long running job should run in asynchronous mode.
+
+## How to configure a job to run asynchronously
+You can configure a job to run asynchronously by specifying wait = FALSE in job options:
+
+```R
+  options <- list(wait = FALSE)
```
Comment (brnleehng) [inline docs/31-long-running-job.md]: Add docs that you can also change the job id?

File: docs/31-long-running-job.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  jobId <- foreach(i = 1:number_of_iterations, .options.azure = options) %dopar% { ... }
+```
+The returned value is the job ID associated with the foreach loop. Use this returned value you can get job status and job result.
+
+## Get job status
+
+getJob returns job metadata, such as chunk size, whether cloud combine is enabled, and packages specified for the job, it also returns task acount in different state
```
Comment (brnleehng) [inline docs/31-long-running-job.md]: account*

File: docs/31-long-running-job.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+```R
+  getJob(jobId)
+  getJob(jobId, verbose = TRUE)
```
Comment (brnleehng) [inline docs/31-long-running-job.md]: Does the verbose print these values out?

What's the output for getJob(jobId)

Comment (zfengms) [reply docs/31-long-running-job.md]: verbose is TRUE by default, so the output for both lines are the same, if this is not desired, we can set verbose to FALSE by default

File: docs/31-long-running-job.md (hunk 2 above)
Comment (zfengms) [reply docs/31-long-running-job.md]: fixed.

File: docs/31-long-running-job.md (hunk 1 above)
Comment (zfengms) [reply docs/31-long-running-job.md]: fixed.

**Pull request merged**